design/TitanOperator Cards.md

```diff
@@ -0,0 +1,30 @@
+##Titan Operator Cards:
+
+Titan's Attack is affected by RPM(~ +.5% damage/RPM)
+Titan starts combat with 25 RPM. This is the baseline. Start of turn the RPM moves closer to the baseline by 1 + 1per10delta from baseline.
+
+Titan's Spell Power is affected by Heat(~+2%/heat)
+Titan starts combat with 2 Heat. He overheats when he has 10 heat.
+
+TheTitanOperator may play any Titanic card as Coolant
+
+Coolant - Titan gains -2 Heat
+
+Lubricate - Titan's cards generate -1 Heat this turn. +8 RPM.
+
+Molten Shield - +2 Heat Gain 1 Shield for each Heat. Overheat: Deal 1/2 current shield amount to all enemies.
+
+Flamespitter - +2 Heat Deal 16(spell) damage to all enemies. Overheat: Deals 20% bonus burn damage.(Burn wounds cannot be healed)
+
+Drill - +1 Heat +10 RPM Deal 1x damage to an enemy. Titan gets +1 Armor this turn.
+
+Adjustments - -2 Heat +10 RPM .5x Defense.
+
+Tinker - -1 Heat +10 RPM baseline (for the rest of the fight), +1 Armor.
+
+Repairs - -2 Heat heal 15-1/Heat.
```

Comment:
How much healing happens? What is the order of operations?

Reply:
E.g. Titan has 5 Heat.  Operator plays this card, reduces heat by 2. Heat is 3, so Titan heals for 12.